server/react-admin/src/AdminApp.js

```diff
@@ -16,134 +18,152 @@ import LevelsForm from './levels/LevelsForm'
 import GeographyRoute from './geographies/GeographyRoute'
 import GeographyForm from './geographies/GeographyForm'
 
+import './adminapp.scss'
+const drawerWidth = 280
+
 function AdminApp() {
+  const location = useLocation()
+  const [open, setOpen] = React.useState(true)
```

Comment:
we should default this to false so it doesn't flash open when navigating back when using the browser back button